RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 ------
 * Adding a block from the post title now shows the add block here indicator.
 * Deselect post title any time a block is added
+* Fix loss of center alignment in image captions on Android
 * Fix merging of empty paragraph blocks when added from a post's title
 * Fix loss of center alignment in image captions
```

Comment:
In light of the added note on line 5, we shouldn't be keeping this note on line 7.

Reply:
Good point @mchowning!